symengine/polys/uintpoly_flint.cpp

```diff
@@ -39,18 +38,18 @@ int UIntPolyFlint::compare(const Basic &o) const
     return 0;
 }
 
+static const fz_t zero_poly(0);
```

Comment:
I don't think having this is a good idea. You can use `UIntPolyFlint::from_dict(var, {{}})` instead, wherever this is needed.